unwrap-mongodb-smt/README.md

```diff
@@ -0,0 +1,126 @@
+# Debezium Unwrap SMT Demo
+
+This setup is going to demonstrate how to receive events from MySQL database and stream them down to a PostgreSQL database and/or an Elasticsearch server using the [Debezium Event Flattening SMT](http://debezium.io/docs/configuration/event-flattening/).
```

Comment:
Is there reference to Elasticsearch correct?

Reply:
No :) It's a left-over from the original unwrap example.